presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -103,6 +105,17 @@ public void setQuery(Query query)
         this.query = query;
     }
 
+    public Optional<Explain> getExplainAnalyze()
+    {
+        return explainAnalyze;
+    }
+
+    public void setExplainAnalyze(Explain explainAnalyze)
```

Comment:
Oh, didn't notice that. Ok, how about `Delete` then? That stores the `Delete` statement separately